src/main/java/org/opensearch/flowframework/workflow/WorkflowData.java

```diff
@@ -44,6 +45,19 @@ public WorkflowData(Map<String, Object> content) {
     public WorkflowData(Map<String, Object> content, Map<String, String> params) {
         this.content = Map.copyOf(content);
         this.params = Map.copyOf(params);
+        this.workflowId = "";
```

Comment:
You can directly call the main constructor like:
`this(content, params, "");`

and remove L46-48